R/literal_coercion_linter.R

```diff
@@ -18,32 +18,24 @@ literal_coercion_linter <- function() {
   base_coercers <- xp_text_in_table(
       paste0("as.", c("logical", "integer", "numeric", "double", "character"))
   )
+
   # notes for clarification:
   #  - as.integer(1e6) is arguably easier to read than 1000000L
   #  - in x$"abc", the "abc" STR_CONST is at the top level, so exclude OP-DOLLAR
   #  - need condition against STR_CONST w/ EQ_SUB to skip quoted keyword arguments (see tests)
   #  - for {rlang} coercers, both `int(1)` and `int(1, )` need to be linted
+  xpath_e <- "expr[2][
```

Comment:
better name needed... not_extraction_or_scientific?

Reply:
Good point. Using the suggested name.